mysql/dao.go

```diff
@@ -97,17 +97,16 @@ func (db *Dao) deleteBySessionID(sessionID []byte) (int64, error) {
 }
 
 // delete session by expiration
-func (db *Dao) deleteSessionByExpiration(expiration int64) (int64, error) {
-	lastTime := time.Now().Unix() - expiration
-	return db.Exec(db.sqldeleteSessionByExpiration, lastTime)
+func (db *Dao) deleteExpiredSessions() (int64, error) {
+	return db.Exec(db.sqlDeleteExpiredSessions, time.Now().Unix())
 }
 
 // insert new session
-func (db *Dao) insert(sessionID, contents []byte, lastActiveTime int64) (int64, error) {
-	return db.Exec(db.sqlInsert, gotils.B2S(sessionID), gotils.B2S(contents), lastActiveTime)
+func (db *Dao) insert(sessionID, contents []byte, lastActiveTime int64, expiration int64) (int64, error) {
```

Comment:
Use `time.Duration` instead of `int64`
Note: Propagate this change to postgres and sqlite3, please

Reply:
What about mysql?

Reply:
ack

Reply:
done